### PR TITLE
ISPN-12840 console microcopy edits

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-12840

### CODE REVIEW

**Review by karesti — APPROVED** [review]

File: src/app/assets/languages/en.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
       "add-entry-form-value-tooltip": "The value can contain simple values ...",
```
Comment (oraNod) [inline src/app/assets/languages/en.json:166]: @karesti Hi, I've noticed the tooltips for keys and values for the create entry dialog are incomplete. 

"The key can contain simple values ..." and "The value can contain simple values ..." 

How should we describe this? The values for keys and values depends on the encoding, right?

**Review by karesti — APPROVED** [review]

File: src/app/assets/languages/en.json (hunk 1 above)
Comment (karesti) [reply src/app/assets/languages/en.json:166]: yes, we can add json or protobuf entities, depending on the encoding of the cache

Comment (oraNod) [reply src/app/assets/languages/en.json]: The key can contain JSON or Protobuf values, depending on cache encoding.

**Comment by oraNod** [comment]
@karesti Hey I updated those two strings. Hopefully the suggestions look OK. Cheers.

**Pull request merged**